docs/cloud-native-security/falco-setup.asciidoc

```diff
@@ -0,0 +1,114 @@
+.. Enable JSON output: `json_output: true`
+.. Enable HTTP output, and under `http_output`, for the `url` value, enter the `url:port` where Falcosidekick will listen. For example if Falcosidekick is running on localhost:
++
+```
+http_output:
+  enabled: true
+  url: "http://0.0.0.0:2801/"
+```
+
+[discrete]
+[[falco-config-falco-for-vms]]
+=== Configure Falcosidekick for VMs:
+
+. Refer to Falcosidekick's documentation to https://github.com/falcosecurity/falcosidekick?tab=readme-ov-file#installation[install Falcosidekick].
+. Use the https://github.com/falcosecurity/falcosidekick?tab=readme-ov-file#installation[Falcosidekick on Docker using environment variables] method and set your environment variables as follows:
+.. `ELASTICSEARCH_HOSTPORT`: your {es} endpoint URL, which can be found under **Connection details** on the upper right of the **Integrations** page in {kib}.
+.. `ELASTICSEARCH_INDEX`: the {es} index where you want to store Falco logs. This must match `logs-falco.alerts-*`.
+.. `ELASTICSEARCH_SUFFIX`: The frequency with which you want the {es} index suffix to change. Either `daily`, `monthly`, `annually`, or `none`. 
+.. `ELASTICSEARCH_APIKEY`: The recommended way to authenticate to {es}, by providing an {kibana-ref}/api-keys.html[API key]. Note that support for this environment variable starts with Falcosidekick version 2.30. You can access the latest version on Falcosidekick's https://hub.docker.com/r/falcosecurity/falcosidekick[Docker Hub].
+.. `ELASTICSEARCH_USERNAME` and `ELASTICSEARCH_PASSWORD`: the username and password for an account on your {es} instance. Authentication using these environment variables is not supported on {ecloud}.
+
+For example:
+
+```
+docker run -d -p 2801:2801 
+           -e ELASTICSEARCH_HOSTPORT=https://test-falco.es.us-west2.gcp.elastic-cloud.com 
+           -e ELASTICSEARCH_INDEX=logs-falco.alerts-all 
+           -e ELASTICSEARCH_SUFFIX=none 
+           -e ELASTICSEARCH_APIKEY=XXXXXXXXXXXXX 
+           -e ELASTICSEARCH_MUTUALTLS=true 
+           -e ELASTICSEARCH_CHECKCERT=true falcosecurity/falcosidekick
+```
+
+IMPORTANT: The {es} account used to authenticate Falcosidekick only needs sufficient privileges to create and write to new indices. We recommend following the principle of least privilege when provisioning this account.
+
+After installing and configuring Falcosidekick, restart docker with `sudo systemctl restart falco`. Falcosidekick should start sending alerts to {es}.
+
+
+[discrete]
+[[ingest-falco-setup-falco-kubernetes]]
+== Configure Falco and Falcosidekick for Kubernetes
+
+First, add the Falco https://helm.sh/docs/topics/charts/[Helm charts]:
+
+```
+helm repo add falcosecurity https://falcosecurity.github.io/charts
+helm repo update
+```
+
+Next, install Falco and Falcosidekick using the `falcosecurity/falco` Helm chart with <<falco-config-falco-for-vms, appropriate values>> for each of the `falcosidekick.config.elasticsearch.*` fields:
```

Comment:
appropriate values hyperlink should be below.

https://github.com/falcosecurity/falcosidekick/blob/master/docs/outputs/elasticsearch.md

if you notice, `falcosidekick.config.*` is static and remaining is the settings column value in the above link.